packages/example-codehub/src/codehub-application.ts

```diff
@@ -3,12 +3,42 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Application} from '@loopback/core';
+import {Provider} from '@loopback/context';
+import {Application, DefaultSequence, FindRoute, InvokeMethod,
+  ParsedRequest, Reject, Send, inject} from '@loopback/core';
+import {Logger, PinoLoggerComponent} from '@loopback/logger';
 import {UserController, HealthController} from './controllers';
+import {ServerResponse} from 'http';
+
+const setupLoggerKey = Logger.SequenceActions.SETUP_LOGGER;
+
+class AppSequence extends DefaultSequence {
+  private defaultHandle: Function;
+  private httpLogger: any;
```

Comment:
We are trying to avoid using `any`. What is the the actual type of a logger function? We should use that type.